peachjam/views/document_problem.py

```diff
@@ -0,0 +1,26 @@
+from django.shortcuts import render
+from django.views.generic.edit import FormView
+
+from peachjam.forms import DocumentProblemForm
+
+
+class DocumentProblemView(FormView):
+    template_name = "peachjam/_report_issue_modal.html"
+    form_class = DocumentProblemForm
+
+    def form_valid(self, form):
+        form = self.get_form()
+        if form.is_valid():
```

Comment:
This is known to be true, so you don't need to call it. Otherwise `self.form_valid()` would not be called.

Reply:
See https://ccbv.co.uk/projects/Django/4.2/django.views.generic.edit/FormView/#post -- the FormView calls `form.is_valid()` and then calls `self.form_valid()` or `self.form_invalid()` -- which means your `form_valid` method can be really simple: the form is valid, just send the email and return the response.